sdk/python/featurestore_sample/notebooks/sdk_and_cli/network_isolation/Network Isolation for Feature store.ipynb

```diff
@@ -515,17 +579,26 @@
     "#### Step 1b. Copy the sample data required for this tutorial series to the newly created storage containers."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To copy sample data to the source data storage container, please assign **Storage Blob Data Contributor** and **Contributor** roles to the user identity. The following cell copies sample source data for transactions feature set used in this tutorial from a public storage account to the newly created storage account."
```

Comment:
please assign *Storage Blob Data Contributor** and **Contributor** roles...

Duplicate?

Reply:
Do you have the cli command for the role assignment?

Reply:
No, both Contributor and Storage Blob Data Contributor roles are needed. For the reasons listed here https://learn.microsoft.com/en-us/azure/storage/blobs/assign-azure-role-data-access?tabs=portal#assign-an-azure-role, Reader or Contributor is needed beside Storage Blob Data Contributor role.